content/docs/user-guide/merge-conflicts.md

```diff
@@ -103,27 +103,22 @@ To resolve conflicted `.dvc` files generated by `dvc import` or
 `dvc import-url`, remove the conflicted hashes altogether:
 
 ```yaml
-< < < < < < < HEAD
-md5: 263395583f35403c8e0b1b94b30bea32
-=======
-md5: 520d2602f440d13372435d91d3bfa176
-> > > > > > > branch
 frozen: true
```

Comment:
@efiop another quick Q: this was added accidentally right? As .dvc files no longer have a top `md5` field. Just double checking

Reply:
@jorgeorpinel This is an import file, so they do have the md5 and so this change is wrong :slightly_frowning_face:

Reply:
- [ ] OK np I'll revert it.

Why do import .dvc files need a top md5 though? We may need to document this in https://dvc.org/doc/user-guide/dvc-files-and-directories or in the import(-url) refs. at least. Thanks

Reply:
We probably will be removing it in the future, it has been in discussions for a while now.

Indeed, looks like https://dvc.org/doc/user-guide/dvc-files-and-directories doesn't have imports at all :slightly_frowning_face: